lib/dep_graph.js

```diff
@@ -38,33 +38,33 @@ function createDFS(edges, leavesOnly, result) {
  */
 var DepGraph = exports.DepGraph = function DepGraph() {
   this.nodes = {};
-  this.outgoingEdges = {}; // Node name -> [Dependency Node name]
-  this.incomingEdges = {}; // Node name -> [Dependant Node name]
+  this.outgoingEdges = {}; // Node -> [Dependency Node]
+  this.incomingEdges = {}; // Node -> [Dependant Node]
 };
 DepGraph.prototype = {
```

Comment:
Yes, it's just a question of terminology. Take whatever you like of my suggestions! In this implementation a node is nothing else than its name, hence my suggestions.